Add tracking stations used in launches from Kourou

leudaimon opened the pull request:
I probably did not use the correct naming convention (nor range), but the location is correct.

leudaimon committed:
Add tracking stations used in launches from Kourou

I probably did not use the correct naming convention (nor range), but the location is correct.

PhineasFreak commented:
Yea, the station IDs should probably be ESTRACK (even if they are not listed by ESA as such but they support all their launches from Kourou). And they should be yellow (1,1,0,1) since they are tracking stations. The ranges are fine.

leudaimon committed:
rename stations used for launches from kourou

leudaimon committed:
fix the colour for georgetown station

NathanKell commented:
PhineasFreak you clearly know this stuff better than I, so go ahead and merge when you feel ready :)

PhineasFreak commented:
@NathanKell I do not have write access to this repository.

leudaimon committed:
Update RemoteTech_Settings.cfg

leudaimon committed:
add Libreville tracking station

eggrobin commented on GameData/RealSolarSystem/RemoteTech_Settings.cfg:
```diff
@@ -931,7 +931,60 @@
+	STATION
+	{
+		Guid	  = 5105f5a9-d628-41c6-ad4b-21154e8fc999
+		Name	  = ESTRACK Ascencion
```
Typo (should be Ascension).

eggrobin commented on GameData/RealSolarSystem/RemoteTech_Settings.cfg:
```diff
@@ -954,7 +1007,7 @@
         STATION
         {
-            Guid	  = 5106f5a9-d628-41c6-ad4b-21154e8fc100
+            Guid	  = 5106f5a9-d629-41c6-ad4b-21154e8fc100
```
What's going on here?

eggrobin reviewed (commented):
Do we want to add Galliot (about 20 km southwest of the ELAs, 30 km southwest of the ELS, across the river from Kourou)?
https://goo.gl/maps/AeJAmrBe6yM2

eggrobin commented:
We definitely want Malindi, https://goo.gl/maps/zCpVyYYuLGt.
Together with Galliot, that would give us the whole network, [Galliot→Natal→Ascension→Libreville→Malindi](http://www.cnes-csg.fr/web/CNES-CSG-fr/9793-les-moyens-de-telemesure-et-localisation.php).

leudaimon commented:
@eggrobin I'm on mobile, so I'll take a look at fixing the ascension thing later. 
About the guid , iirc I noticed there were two stations with identical IDs, so I just changed one. 
About Galliot, isn't it too close to kourou to matter?
Malindi is already in the file, just below the place where I added the other ones

eggrobin commented:
Ah yes, I see Malindi. If we're counting Natal, Ascension, and Librevile as ESTRACK, should Malindi be like that too?
As for Galliot, the thing is the Kourou station in that file has a tiny range, whereas Galliot has the same sort of equipment as the others. If Kourou is in sight of Galliot in-game, perhaps the Kourou station could be removed and we could just keep Galliot, I'm pretty sure Galliot has telemetry from the start IRL: see [this document about VA226](http://www.airbus.com/content/dam/corporate-topics/publications/press-release/dossier-de-vol-v226-en.pdf), section 8, there is no acquisition Galliot.

PhineasFreak commented:
@eggrobin It would probably be better to keep the Kourou station since every RSS launch site is also a tracking station.

eggrobin commented:
A random thought from looking at this file, the comment at the beginning of the file is incorrect, since @NathanKell [changed the value from 250 km to 5000 km](https://github.com/leudaimon/RealSolarSystem/commit/fbc0ef295dceb998d6aa9f826df795f53739c5a9) without updating [the comment introduced](https://github.com/leudaimon/RealSolarSystem/commit/ddbaa89c065207e6c8f3892c6060218e83066439) by @jbengtson.

eggrobin commented:
@PhineasFreak, that makes sense, but then we still need the longer-range station of Galliot (with a known dish, rather than just "is a launch site so 5000 km").

PhineasFreak commented:
@eggrobin Oh, that's OK, more stations are always better! Probably misunderstood your comment...

leudaimon committed:
Add Galliot tracking station

also fixed other naming details in Ascension and Malindi

The ranges for all these new tracking stations is not accurate according to antenna sizes, more of a guess.

eggrobin commented:
Looks good to me

NathanKell commented:
Merging. Also @PhineasFreak your perms are fixed. :)
